modules/graphql_twig/graphql_twig.module

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/**
+ * @file
+ * GraphQL Twig hook implementations.
+ */
+
+use Drupal\graphql\GraphQL\Execution\Processor;
+use Drupal\graphql_twig\QueryAssembler;
+
+/**
+ * Implements hook_theme_registry_alter().
+ */
+function graphql_twig_theme_registry_alter(&$themeRegistry) {
+  $queryRegex = '/\{#graphql-query\n(?<query>.*)\n#\}\n/s';
+  $fragmentRegex = '/\{#graphql-fragment\n(?<fragment>.*)\n#\}\n/s';
+  $assembler = new QueryAssembler();
+  $queries = [];
+
+  foreach ($themeRegistry as $hook => $info) {
+    if (!array_key_exists('template', $info)) {
```

Comment:
This condition is not 100%. 'template' is the default and can be omitted. This should check for "function" instead. 

https://www.drupal.org/node/2231673